script/build

```diff
@@ -14,7 +14,9 @@ windows=
 
 setup_gopath() {
   TMPDIR="${LOCALAPPDATA:-$TMPDIR}"
-  TMP_GOPATH="${TMPDIR:-/tmp}/go"
+  MY_TMPDIR=${TMPDIR:-/tmp}
```

Comment:
There is no need to create a new variable `MY_TMPDIR` here. You can just override TMPDIR.